exec-ui/src/app/modules/metrics/modules/metrics/traceability/traceability.configuration.ts

```diff
@@ -0,0 +1,8 @@
+export class TraceabilityConfiguration {
+  public static identifier = 'traceability';
+  public static previewHeading = 'Traceability';
+  public static detailHeading = 'Traceability';
+  public static graphHeading = 'Traceability';
+  public static buildingBlockLabel = 'Traceability';
+  public static description = 'Traceability is the percentage of accepted stories with appropriate test cases. It establishes links between requirements and tests which helps in tracing back to the expected business functionality.';
```

Comment:
Add the way the metric is calculated:
% Test Traceability = # of feature files tagged with Story IDs from test execution results / Total # of feature files found in Git